00-open-a-shapefile.Rmd

```diff
@@ -136,19 +134,24 @@ shapefiles include
 
 * a polygon shapefile representing our study area boundary, 
 * a line shapefile representing roads and streams, and 
-* a point shapefile representing the location of a [flux tower](http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements) located at the NEON 
-Harvard Forest field site.
+* a point shapefile representing the location of a [flux tower](http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements) 
+located at the NEON Harvard Forest field site.
 
```

Comment:
Any time we reference a field site, let's make sure we link to it using code as follows: I am not using markdown because you can't force markdown to open in a new window.

<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest Field Site</a>